Name: Improved Edited Video Filenames
Version: 1.4.0-1
Author: Jason Robitaille
Description: Will output edited videos in the format <filename>_edit.mp4  (MIT license)

--- .orig/usr/palm/frameworks/private/metascene.videos/submission/76/javascript/assistants/browser-assistant.js
+++ /usr/palm/frameworks/private/metascene.videos/submission/76/javascript/assistants/browser-assistant.js
@@ -185,6 +185,9 @@
 		} else {
 			// show details scene
 			params.captured = this.query.captured;
+			params.destination = params.target.substring(0,
+					params.target.lastIndexOf(".")) + "_edit" +
+					params.target.substring(params.target.lastIndexOf("."));
 			metascene.Push(this.controller.stageController, exports.Details, params);
 		}
 	}
